src/Tizen.NUI/src/public/Theme/ThemeManager.cs

```diff
@@ -161,6 +161,21 @@ public static void ApplyTheme(Theme theme)
                 newTheme.Id = "NONAME";
             }
 
+            if(!string.IsNullOrEmpty(newTheme.SmallBrokenImageUrl))
```

Comment:
In case the user sets the `BrokenImageUrl` as an empty string (= invalid url) intentionally, don't we have to pass that data to the DALi?